document deployments ordering (#1841)
mendersoftware/mender-docs: 02.Overview/05.Deployment/docs.md

Only accepted devices can be part of a Deployment, and any given device finishes the Deployment once.
Mender also maintains the order of the Deployments, so the oldest Deployment is the one a given device gets first.

> Maintaining the order of the Deployments also means, that the device will never get the Deployment older than the one recently installed on the device.

Reviewer: * Remove comma after "means"
* "..the one **most** recently installed..."

> Maintaining the order of the Deployments also means that the device will never get the Deployment older than the one most recently installed on the device.

Reviewer 2: ```suggestion
Maintaining the order of the Deployments also means that the device will never get a Deployment created before the most recently installed one.
```

Author: done (both, the text and the diagram)

> 

Reviewer 3: 02.Overview/05.Deployment/selectDeploymentForDeviceAlgorithm.png comment: the picture only changed the first block, right? (my png diff is not perfect :> and had to clone and open the file) suggested change: "get oldest, not finished deployment, but " -> "get oldest, _pending_ deployment, but ". just trying to be picky.

Reviewer 3: also: baseline of "matching device?" text does not match the base line of "match artifact" the fonts are different, right? it may have been a preexisting problem.

Author: done with the first comment;
about the second comment - in my editor both are using same font
